# time_pilot.vhd: vcnt reload 0x0FC -> 0x0F8, 264 lines per frame (60.61 Hz) per MAME
# konami/pooyan.cpp set_raw (same Konami video); the core had 260 lines (61.54 Hz).
# Applies after time_pilot_single_domain.patch.
--- a/vhdl_time_pilot_rev_0_0_2017_11_05/rtl_dar/time_pilot.vhd
+++ b/vhdl_time_pilot_rev_0_0_2017_11_05/rtl_dar/time_pilot.vhd
@@ -292,7 +292,7 @@
           if hcnt = "101111" then -- char from #0 to #47 (one line)
             hcnt <= "000000";
             if vcnt = '1'&X"FF" then
-              vcnt <= '0'&X"FC";
+              vcnt <= '0'&X"F8"; -- 248: 264 lines (60.61 Hz) per MAME konami/pooyan.cpp
             else
               vcnt <= vcnt + '1';
             end if;
